RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -125,6 +125,8 @@ private extension CustomerCenterView {
                     ManageSubscriptionsView(screen: screen,
                                             customerCenterActionHandler: viewModel.customerCenterActionHandler)
                 }
+            } else if let screen = configuration.screens[.management] {
+                WrongPlatformView(screen: screen)
```

Comment:
Hmm are we passing the management screen to the `WrongPlatformView`? Seems wrong to me... As in, it should be a separate screen right?

Reply:
We are not adding another screen to the frontend yet, so reusing that title which is for the "Users with an active subscription", which also applies to this screen.

We should build this screen dynamically as well, and I am thinking we should actually reuse the Management screen for it (filtering out things that don't make sense, like asking for a refund), what do you think? I would do that in another PR though

Reply:
Yes it's mainly the custom URLs and the soon to come contact support one which is hardcoded in this screen atm. I don't know what to do with the manage subscriptions button, which is not a path either but doesn't make sense in the regular management screen